bolt/test/AArch64/fp-reg-spill.s

```diff
@@ -0,0 +1,19 @@
+# Check that we correctly handle arm64 fp register spills in
+# bolt when we are processing jump tables.
```

Comment:
Sorry for jumping in late after the change landed. This test checks that BOLT doesn't crash, but it doesn't really check for the correct handling of the jump table. You'll need to add `FileCheck` checks for `--print-cfg` dumps or other checks for the output.

Reply:
The idea behind the test was not to check the jump table (although it would be nice), but the previous bolt assertion during analyzation. So the test is good-enough I think